### PR TITLE
LFH-72: Categories changed to navigation tabs

### PR DESCRIPTION
**Pull request by KG700**
Updated Categories component:
- Used Tabs component from Material ui to reformat component

image

### CODE REVIEW

File: src/Categories/index.js [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
-    fontWeight: "bold"
-  }
+    borderBottom: `${cyan[300]} 4px solid`,
+  },
 }));
 
 export const Categories = ({ handleFilterChange, categoryList }) => {
   const classes = useStyles();
-  const [categoryTitle, setCategoryTitle] = useState(null);
+  const [categoryIndex, setCategoryIndex] = useState(0);
```
Comment (Caitlin-cooling) [inline src/Categories/index.js:22]: Sorry I know you were following my bad example but could we call  this `selectedCategoryIndex`? Just makes it a bit clearer

File: src/Categories/index.js [hunk 2]
```diff
@@ -1,59 +1,60 @@
...
-      </Typography>
-      <List>
-        <ListItem button onClick={handleCategorySelection} value={null}>
-          <ListItemText
-            primary="All"
-            classes={{ primary: categoryTitle === null ? classes.selected : "" }}
+      <Tabs
+        value={categoryIndex}
+        onChange={handleCategorySelection}
+        indicatorColor="primary"
```
Comment (Caitlin-cooling) [inline src/Categories/index.js:34]: probably don't need this prop as we're hiding the indicator right?

File: src/Categories/index.js [hunk 3]
```diff
@@ -1,59 +1,60 @@
...
-      </List>
+      </Tabs>
     </div>
   );
 };
 
 Categories.propTypes = {
   handleFilterChange: PropTypes.func,
-  categoryList: PropTypes.array
+  categoryList: PropTypes.array,
```
Comment (Caitlin-cooling) [inline src/Categories/index.js:59]: This should be throwing a linting warning, could we remove please and let me know if it's not I'll look at fixing the config

File: src/Home/index.js [hunk 4]
```diff
@@ -18,48 +24,48 @@ const drawerWidth = 300;
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    display: "flex"
+    display: "flex",
```
Comment (Caitlin-cooling) [inline src/Home/index.js:27]: Same with the danlging commas in this file

File: src/Categories/index.js (hunk 1 above)
Comment (Caitlin-cooling) [reply src/Categories/index.js:22]: Actually this wreaks havoc with the API calls - ignore

File: src/Categories/index.js (hunk 3 above)
Comment (KG700) [reply src/Categories/index.js:59]: Yep, it does throw a linting warning. When I ran `npm run eslint` there were actually quite a few trailing comma linting warnings - I think my vscode must have done this. Will need to fix my settings somehow.

**Pull request merged**